### PR TITLE
NMS-14699: Export poller configuration

### PR DESCRIPTION
**Pull request by dherdt**
Jira: https://issues.opennms.org/browse/NMS-14699

### CODE REVIEW

File: plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.List;
+
+import org.opennms.integration.api.v1.config.poller.Monitor;
+import org.opennms.integration.api.v1.config.poller.Package;
+import org.opennms.integration.api.v1.config.poller.PollerConfigurationExtension;
+import org.opennms.integration.api.xml.ClasspathPollerConfigurationLoader;
+
+public class VelocloudPollerConfigurationExtension implements PollerConfigurationExtension {
+        final PollerConfigurationExtension pollerConfiguration = new ClasspathPollerConfigurationLoader(VelocloudPollerConfigurationExtension.class, "",
+                "velocloud-poller-configuration.xml").getPollerConfiguration();
```
Comment (fooker) [inline plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java:40]: There is already a `poller-configuration.xml` in resources. Should it be renamed and does it have the right syntax/format?

**Review by fooker — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java (hunk 1 above)
Comment (dherdt) [reply plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java:40]: in Integration-API in poller-configuration.xsd is specified that `package`has to have one `rrd

Comment (dherdt) [reply plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java:40]: Which ```rra```s should be specified there?

Comment (dherdt) [reply plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java:40]: ...and at least one ```downtime``` is also needed

Comment (fooker) [reply plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java:40]: lets use the default from the horizon poller-configuration for now

**Review by fooker — APPROVED** [review]

**Pull request merged**